### DISCUSSION TITLE
Dagesh Chazaq and syllable separator

### DISCUSSION THREAD

**Discussion by m-yac** [charlesLoder/hebrew-transliteration#68] [Ideas]
Currently, if `DAGESH_CHAZAQ` is `true` and `SYLLABLE_SEPARATOR` has a value we get:
```javascript
heb.transliterate("שַׁבָּת", { SYLLABLE_SEPARATOR: "-" }) === "ša-bbāt"
```
but I think it really ought to be `"šab-bāt"`. Thoughts?

**Comment by johnlockejrr** [comment]
For sure has to be “šab-bāt” because DAGESH_CHAZAQ is a syllable separator.

**Reply by charlesLoder** [reply]
100% should be that. That was another issues blocking #45
